lib/knife-clc/bootstrap/config_options.rb

```diff
@@ -24,6 +24,11 @@ def self.attach(command_class)
               :long => '--bootstrap-platform PLATFORM',
               :description => 'Assume bootstrapping server platform as windows or linux. Derived automatically by default',
               :on => :head
+
+            option :bootstrap_protocol,
```

Comment:
Do we need this option? For windows we support only WinRM. For linux - only SSH.